server/models/Development.js

```diff
@@ -0,0 +1,42 @@
+const keystone = require('keystone');
+const { Name } = require('keystone/lib/fieldTypes');
+const Types = keystone.Field.Types;
```

Comment:
I also have a suggestion, you can change this to: ``` const { Types } = keystone.Field```. Helps your code stay cleaner which is very important.

If you want to understand it better, this is called destructuring attribution. You can read more here: https://www.alura.com.br/artigos/es6-desestruturando-objetos. 

Same recommendation for your banner section branch.